docs/guides/quick-answers/linux/how-to-install-selinux-on-ubuntu-18-04/index.md

```diff
@@ -141,5 +140,143 @@ If you do not see the this entry, open the port with the following command:
     sudo semanage port -a -t ssh_port_t -p tcp 22
     {{</ note >}}
 
+## Enabling Policies SELinux Policies On Ubuntu 18.04 
+
+SELinux comes with a set of pre-built policies to handle requests that drive security. These policies determine if any request should be allowed to be processed. 
+
+For example, to enable a policy that allows MySQL requests through SELinux. MySQL’s documentation recommends that you disable SELinux to handle such requests, but there are other and better options. 
+
+
+### Enabling SELinux Policy for MySQL Requests
+
```

Comment:
May be a good idea to add installing MySQL on a Linode in the before you begin section. Because the user cannot execute these commands on a Linode that does not have MySQL/MariaDB.